package.json

```diff
@@ -33,7 +33,7 @@
     "eslint-plugin-compat": "^2.5.1",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.0"
+    "eslint-plugin-react": "7.11.0"
```

Comment:
7.12.1 was released last night, are the issues fixed?

Reply:
Not sure 😄 I found this bug a few days ago. I will check

Reply:
😭 tested out the newest release and there are still issues with the `reportUndeclaredPropTypes` rule. Logged an issue to eslint-plugin-react: https://github.com/yannickcr/eslint-plugin-react/issues/2108.

Reply:
Nice work looking into this!